content/en/docs/contribute/review/reviewing-prs.md

```diff
@@ -153,6 +159,18 @@ When reviewing, use the following as a starting point.
 
 ### Other
 
-For small issues with a PR, like typos or whitespace, prefix your comments with `nit:`.
-This lets the author know the issue is non-critical.
-
+- Watch out for [trivial edits](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits);
+  if you see a change that you think is a trivial edit, please point out that policy
+  (it's still OK to accept the change if it is genuinely an improvement).
+- Encourage authors who are making whitespace fixes to do
+  so in the first commit of their PR, and then add other changes on top of that. This
+  makes both merges and reviews easier. Watch out especially for a trivial change that
+  happens in a single commit along with a large amount of whitespace cleanup
+  (and if you see that, encourage the author to fix it).
+
+As a reviewer, if you identify small issues with a PR that aren't fundamental to the meaning, such as typos or incorrect whitespace, prefix your comments with `nit:`.
+This lets the author know that this part of your feedback is non-critical.
+If you are considering a pull request for approval and all the remaining feedback is
+marked as a nit, you can merge the PR anyway. In that case, it's often useful to open
+an issue about the remaining nits. You can usually mark that new issue as a “Good
+First Issue”.
```

Comment:
```suggestion
As a reviewer, if you identify small issues with a PR that aren't essential to the meaning,
such as typos or incorrect whitespace, prefix your comments with `nit:`.
This lets the author know that this part of your feedback is non-critical.
If you are considering a pull request for approval and all the remaining feedback is
marked as a nit, you can merge the PR anyway. In that case, it's often useful to open
an issue about the remaining nits. You can usually mark that new issue as a “Good
First Issue”.
```

The guidance about good-first-issue I'm not sure about - do we have a guide about making an issue a GFI in sig-docs? I thought you had to make a commitment to providing support on that issue, and that you had to provide resources that an assignee could use to complete the issue without...issues (heh)?